db/migrate/20200506182951_add_value_to_metric_name_enum.rb

```diff
@@ -0,0 +1,16 @@
+class AddValueToMetricNameEnum < ActiveRecord::Migration[6.0]
+  def up
+    execute <<-SQL
+      ALTER TYPE metric_name ADD VALUE 'merge_time';
+    SQL
+  end
+
+  def down
+    execute <<-SQL
+      ALTER TYPE metric_name RENAME TO metric_name_old;
+      CREATE TYPE metric_name AS ENUM ('review_turnaround', 'blog_visits');
+      ALTER TABLE metrics ALTER COLUMN name TYPE metric_name USING name::text::metric_name;
+      DROP TYPE metric_name_old;
```

Comment:
this SQL doesn't look like the inverse of the `up` method. what am I missing?

Reply:
You can't just delete an item from an ENUM, but you can add. So the only way to "delete" from an ENUM is changing the name of the current ENUM and create a new one with the old values.